ros-image_recognition_pytorch/install.bash

```diff
@@ -0,0 +1,10 @@
+#! /usr/bin/env bash
+
+url="https://github.com/Nebula4869/PyTorch-gender-age-estimation/raw/038331d26fc1fbf24d00365d0eb9d0e5e828dda6/models-2020-11-20-14-37/best-epoch47-0.9314.onnx"
+dest=~/pytorch/models
```

Comment:
```suggestion
dest=~/data/pytorch_models
```

Reply:
Also requires a change in the node and downloader script

Reply:
True, commented there too